tmcprototype/CspSubarrayLeafNode/CspSubarrayLeafNode/CspSubarrayLeafNode.py

```diff
@@ -86,7 +86,7 @@ def commandCallback(self, event):
     # Device Properties
     # -----------------
     CspSubarrayNodeFQDN = device_property(
```

Comment:
Property name should be CspSubarrayFQDN.

Reply:
Done.

Reply:
Commit ID: 9ab1a8bbeec93788ee716d8c14cabd8cb9e7a1bd